.github/workflows/welcome-new-contributors.yml

```diff
@@ -0,0 +1,47 @@
+name: Welcome new contributors to the repo
+
+on:
+  issues:
+    types:
+      - opened
+
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  welcome-message:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            # Welcome!
+            Welcome to the ICDS repo, thank you for raising an issue!
```

Comment:
Hey. Thank you for your contribution. Since this workflow will also be added to the ic-ui-kit repo, could you change this and the PR message to specify the exact repo contributors are adding to?

Reply:
Hi, I've updated this to state the repo name. Do you want me to open a PR against the `ic-ui-kit` repo as well, or is going to be covered under a separate issue?

Reply:
Hey! Don't worry about the `ic-ui-kit` repo, we have a separate issue for that repo